src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -14,34 +14,33 @@ The .NET Foundation licenses this file to you under the MIT license.
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <PropertyGroup Condition="'$(TargetArchitecture)' == 'wasm'">
-    <NativeCodeGen>llvm</NativeCodeGen>
-  </PropertyGroup>
```

Comment:
Doesn't this (and below) completely remove the build support for LLVM/WASM?

Reply:
oh yeah, thanks, will revert that